R/createReport.R

```diff
@@ -552,11 +553,14 @@ createReport = function(txt_folder = NULL, mztab_file = NULL, yaml_obj = list(),
     ##
     ## missed cleavages per Raw file
     ##
-    if (!is.null(df_evd)) {
+    # df_evd can be NULL; that's no problem
```

Comment:
check indentation  for both `lst_qcMetrics[["qc ... ` (looks like two extra spaces in front?)